src/test/scala/examples/SyncReadMemTest.scala

```diff
@@ -0,0 +1,66 @@
+// See README.md for license details.
+
+package examples
+
+import org.scalatest.{FreeSpec, Matchers}
+import chisel3._
+import chisel3.iotesters.PeekPokeTester
+import logger.LazyLogging
+
+class HasSyncReadMem extends MultiIOModule {
+  val readAddr  = IO(Input(UInt(16.W)))
+  val readData  = IO(Output(UInt(16.W)))
+  val writeAddr = IO(Input(UInt(16.W)))
+  val writeData = IO(Input(UInt(16.W)))
+  val writeEnable = IO(Input(Bool()))
+
+  val mem = SyncReadMem(10, UInt(16.W))
+
+  readData := mem(readAddr)
+  when(writeEnable) {
+    mem(writeAddr) := writeData
+  }
+}
+
+class SyncReadMemTest extends FreeSpec with Matchers {
+  "peekAt and pokeAt should work with treadle" in {
+    iotesters.Driver.execute(
+      Array(
+        "--backend-name",
+        "treadle",
+        "--target-dir",
+        "test_run_dir/sync_read_mem_test_treadle",
+        "--top-name",
+        "SyncReadMem",
+      ),
```

Comment:
Fails on Scala 2.11 - trailing comma.

Reply:
Arghh, is there anyway to catch this

Reply:
Yes.
```bash
$ sbt +test
```
 will compile and test on all `crossScalaVersions`.